docs/openapi/open-api.yml

```diff
@@ -56,6 +78,53 @@ paths:
                     mTeam: './mteam-league.yml#/components/schemas/MTeamLeagueResponse'
                     mRoster: './mroster-league.yml#/components/schemas/MRosterLeagueResponse'
 
+  /{sportId}/seasons/{seasonYear}/segments/0/leagues/{leagueId}/transactions:
+    post:
+      summary: 'Submit lineup changes'
+      description:
+        POST request to submit lineup changes
+      parameters:
+        - name: sportId
+          in: path
+          description: >
+            The string based id of the fantasy sport.
+            * nfl - ffl
+            * nba - fba
+            * nfl - fhl
+            * mlb - mlb
+          required: true
+          schema:
```

Comment:
Pull this schema out into the `components/schemas` since it is reused multiple times.